releaser/src/releaser.rs

```diff
@@ -0,0 +1,227 @@
+        };
+
+        for package in self.packages.iter() {
+            let old_version = &versions[&package.name];
+            if old_version < &bump_version {
+                println!(
+                    "Bumping {} package {} from {} to {}",
+                    style(&package.package_type).bold().red(),
+                    style(&package.name).bold().cyan(),
+                    style(&old_version).bold().magenta(),
+                    style(&bump_version).bold().green()
+                );
+                package.set_repo_version(&bump_version)?;
+            } else {
+                println!(
+                    "{} package {} is already at {}",
+                    style(&package.package_type).bold().red(),
+                    style(&package.name).bold().cyan(),
+                    style(&bump_version).bold().yellow()
+                );
+            }
+        }
+
+        // Bump dependencies
+        for package in self.packages.iter() {
+            let package_type = package.package_type;
+            let Some(deps) = deps.get(&package_type) else {
+                continue;
+            };
+            println!(
+                "Updating dependencies for {} package {}",
+                style(&package.package_type).bold().red(),
+                style(&package.name).bold().cyan()
+            );
+            let result = package
+                .package_type
+                .get_package_manager()
+                .update_dependencies(&package.path, deps, &bump_version)
+                .context("Updating dependencies")?;
+            if result {
+                println!(
+                    "Updated dependencies for {} package {}",
+                    style(&package.package_type).bold().red(),
+                    style(&package.name).bold().cyan()
+                );
+            }
+        }
+
+        // Update lockfiles
+        for package in self.packages.iter() {
+            if package.package_type == PackageType::Python {
+                continue;
+            }
+            println!(
+                "Updating lockfile for {} package {}",
+                style(&package.package_type).bold().red(),
+                style(&package.name).bold().cyan()
+            );
+            package
+                .package_type
+                .get_package_manager()
+                .update_lockfile(&package.path)?;
+        }
+
+        // Prompt to confirm commit to git.
+        let confirm = Confirm::new()
+            .with_prompt("Commit changes to git?")
+            .default(false)
+            .interact()?;
+        if !confirm {
+            return Ok(());
+        }
+
+        // Check out a branch
+        let branch_name = format!("release/{}", bump_version);
+        self.git.checkout_new_branch(&branch_name)?;
+
+        // Commit changes
+        self.git
+            .commit_all(&format!("Bump version to {}", bump_version))?;
+
```

Comment:
We should also add a git tag to this commit like `v0.3.0`

Reply:
The tag is automatically applied when we release, as I understand (e.g. I used this script to deploy v0.3.0 and it shows up here https://github.com/jamsocket/y-sweet/tags)